pkg/syncer/metadatasyncer.go

```diff
@@ -551,6 +573,198 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 	return nil
 }
 
+// startTopologyCRInformer creates and starts an informer for CSINodeTopology custom resource.
+func startTopologyCRInformer(ctx context.Context, cfg *restclient.Config) error {
+	log := logger.GetLogger(ctx)
+	// Create an informer for CSINodeTopology instances.
+	dynInformer, err := k8s.GetDynamicInformer(ctx, csinodetopologyv1alpha1.GroupName,
+		csinodetopologyv1alpha1.Version, csinodetopology.CRDPlural, metav1.NamespaceAll, cfg, true)
+	if err != nil {
+		return logger.LogNewErrorf(log, "failed to create dynamic informer for %s CR. Error: %+v",
+			csinodetopology.CRDSingular, err)
+	}
+	csiNodeTopologyInformer := dynInformer.Informer()
+	csiNodeTopologyInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			topoCRAdded(obj)
+		},
+		UpdateFunc: func(oldObj interface{}, newObj interface{}) {
+			topoCRUpdated(oldObj, newObj)
+		},
+		DeleteFunc: func(obj interface{}) {
+			topoCRDeleted(obj)
+		},
+	})
+	// Start informer.
+	go func() {
+		log.Infof("Informer to watch on %s CR starting..", csinodetopology.CRDSingular)
+		csiNodeTopologyInformer.Run(make(chan struct{}))
+	}()
+	return nil
+}
+
+func addLabelsToTopologyVCMap(ctx context.Context, nodeTopoObj csinodetopologyv1alpha1.CSINodeTopology) {
```

Comment:
Can you add function document for `addLabelsToTopologyVCMap`?

Reply:
Done